monai/networks/nets/magnus.py

```
@@ -0,0 +1,775 @@
+
+from collections.abc import Sequence
+from typing import Optional, Union
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+from monai.networks.blocks import Convolution, UpSample
+from monai.networks.layers.utils import get_act_layer, get_norm_layer
+
+__all__ = ["MAGNUS", "CNNPath", "TransformerPath", "CrossModalAttentionFusion", "ScaleAdaptiveConv"]
```

Ruff (0.14.13)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)